scout/constants/clnsig.py

```diff
@@ -30,6 +30,7 @@
     "likely_benign": 3,
     "conflicting_interpretations_of_pathogenicity": 8,
     "conflicting_interpretations": 8,
+    "conflicting_classifications_of_pathogenicity": 8,
```

Comment:
Thank goodness the variant is built using this reverse map and in the end gets the number as value instead of the string itself!

<img width="319" alt="image" src="https://github.com/user-attachments/assets/2a07a404-5d84-4f7e-91ae-7a0177f792b4">

This fix should be enough! 👍🏻

Reply:
Yes, this is a fortunate design. They even had the numbers directly in the file way back when. 🙄 But I guess that made it a bit less human readable.